### ISSUE TITLE
MK8 Thwomp temple always crashing vulkan randomly

### ISSUE THREAD

**Issue by Golem642** [cemu-project/Cemu#1135]
### Current Behavior

When i try to play the Thwomp temple course in MK8, no matter what it will always crash after a random amount of time. It can even crash during the intro cutscene. The game just freezes, vulkan bugs out entirely, makes my screen go black, then comes back up and Cemu finally quits because of a vulkan error

### Expected Behavior

I'd expect to play the course normally without any problems. OpenGL works but there are way too many ghosting effects and lighting issues it's not even enjoyable

### Steps to Reproduce

- Open Cemu
- Open MK8 (have it installed first)
- Go to any kind of mode where you can play the courses (even grand prix)
- Play the Thwomp temple course
- After some random time, it will crash. You never know when but it will. I never got to finish the course with vulkan

### System Info (Optional)

OS: Windows 10 22H2
GPU: Intel Iris Xe

### Emulation Settings (Optional)

Vulkan with async, no vsync, bicubic upscale, bilinear downscale, keep aspect ratio scaling

### Logs (Optional)

Gonna submit it right after, i just opened Cemu to check my graphics settings and it overwrote my logs. Since i don't want to rewrite everything here because of a graphic crash (yes it sometimes resets the browser view) i'm sending this now

**Comment by Golem642** [comment]
[log.txt](https://github.com/cemu-project/Cemu/files/14736447/log.txt)

**Comment by goeiecool9999** [comment]
The log says ``Msg: failed to submit command buffer. Error -4`` -4 is the vulkan error code VK_ERROR_DEVICE_LOST. This indicates a problem at with the drivers.

**Comment by Golem642** [comment]
for reference, my drivers are up to date. i checked for other vulkan errors like last time but didn't find anything, so perhaps a certain command that doesn't exist is being ran ? like i said, it really only does this on the thwomp temple course, and i think it happened one other time on splatoon but never again since.
Also, somehow cemu keeps recompiling the same pipelines over and over again ? like they are loaded and they already exist for sure (no ghosting effect) but it always says "9 new pipelines compiled" when i boot up the game or even "61 new pipelines compiled" when i load thwomp temple. Even after i load it, quit the course, go back to the main menu and exit cemu manually. Those seem to always be recompiled for some reason

**Comment by inklingboy97** [comment]
This seems like the drivers are corrupted in some way, have you tried reinstalling them?

**Comment by Golem642** [comment]
> This seems like the drivers are corrupted in some way, have you tried reinstalling them?

I just did, same problem. Although it recompiled some of the courses pipelines, when i got to thwomp temple, my Luigi had those black gaps (which apparently is normal) and I couldn't even start the race that it crashed

**Comment by Golem642** [comment]
Recently got a new driver update, I did it and it still doesn't work. I really don't know where the problem is coming from. It also crashed upon first-time loading a map in Splatoon, but only after a few first-time loaded maps

**Comment by Exzap** [comment]
Closing this issue since it's stale. Intel Xe GPUs have gotten quite a few driver updates so there is a good chance this issue has resolved itself? If you still run into the problem feel free to open a new issue

**Comment by Golem642** [comment]
> Closing this issue since it's stale. Intel Xe GPUs have gotten quite a few driver updates so there is a good chance this issue has resolved itself? If you still run into the problem feel free to open a new issue

I don't have the PC I initially used for this but I still have all of the files, I'll try later to see if the problem is still here

**Comment by Golem642** [comment]
Can confirm the problem disappeared, along with some other overglow problems (not all tho)
I'm now using an integrated AMD Radeon 610M GPU, so maybe it had to do with Intel? I had a "device lost" problem on a few games and emulators
